src/RadixDlt.NetworkGateway.PostgresIntegration/Services/ValidatorQuerier.cs

```diff
@@ -101,26 +96,38 @@ public ValidatorQuerier(ReadOnlyDbContext dbContext, IDapperWrapper dapperWrappe
             .AnnotateMetricName("GetValidators")
             .ToDictionaryAsync(e => e.Id, e => e.Address, token);
 
-        var cd = new CommandDefinition(
-            commandText: @"
-SELECT
-    validator_entity_id AS ValidatorEntityId,
-    SUM(proposals_made)::bigint AS ProposalsMadeSum,
-    SUM(proposals_missed)::bigint AS ProposalsMissedSum,
-    COUNT(proposals_made) AS EpochsActiveIn
-FROM validator_emission_statistics
-WHERE validator_entity_id = ANY(@validatorEntityIds) AND epoch_number BETWEEN @epochFrom AND @epochTo
-GROUP BY validator_entity_id;",
-            parameters: new
-            {
-                validatorEntityIds = validators.Keys.ToList(),
-                epochTo = ledgerState.Epoch,
-                epochFrom = fromLedgerState?.Epoch ?? 0,
-            },
-            cancellationToken: token);
+        var validatorIds = validators.Keys.ToList();
+        var epochFrom = fromLedgerState?.Epoch ?? 0;
+        var epochTo = ledgerState.Epoch;
 
-        var validatorUptime = (await _dapperWrapper.QueryAsync<ValidatorUptimeViewModel>(_dbContext.Database.GetDbConnection(), cd, "GetValidatorEmissionStatistics"))
-            .ToDictionary(e => e.ValidatorEntityId);
+        var validatorUptime = await _dbContext
```

Comment:
you're missing metric annotation `AnnotateMetricName` I think we've missed these in account locker queries as well.

Reply:
Good catch! Added.